ui/app/components/mount-backend-form.js

```diff
@@ -108,14 +108,30 @@ export default Component.extend({
 
   actions: {
     onKeyUp(name, value) {
-      this.mountModel.set('path', value);
-      this.mountModel.validations.attrs.path.isValid
-        ? set(this.validationMessages, 'path', '')
-        : set(this.validationMessages, 'path', this.mountModel.validations.attrs.path.message);
-
-      this.mountModel.validate().then(({ validations }) => {
-        this.set('isFormInvalid', !validations.isValid);
-      });
+      // validate path
+      if (name === 'path') {
+        this.mountModel.set('path', value);
+        this.mountModel.validations.attrs.path.isValid
+          ? set(this.validationMessages, 'path', '')
+          : set(this.validationMessages, 'path', this.mountModel.validations.attrs.path.message);
+
+        this.mountModel.validate().then(({ validations }) => {
+          this.set('isFormInvalid', !validations.isValid);
+        });
+      }
+      // check maxVersions is a number
```

Comment:
add validations for mounting a secret engine. this is relevant for kv2 specifically as we now show maxVersion on the config screen. (we post to both sys/mount and now secret/config)